mopidy_musicbox_webclient/static/js/controls.js

```diff
@@ -357,40 +357,18 @@ function doSingle() {
  * Use a timer to prevent looping of commands  *
  ***********************************************/
 function doSeekPos(value) {
-    var val = $("#trackslider").val();
-    newposition = Math.round(val);
-    if (!initgui) {
-        pausePosTimer();
-        //set timer to not trigger it too much
-        clearTimeout(seekTimer);
-        $("#songelapsed").html(timeFromSeconds(val / 1000));
-        seekTimer = setTimeout(triggerPos, 500);
-    }
-}
-
-function triggerPos() {
     if (mopidy) {
-        posChanging = true;
-        mopidy.playback.seek({'time_position': newposition});
-        resumePosTimer();
-        posChanging = false;
+        mopidy.playback.seek({'time_position': Math.round(value)});
     }
 }
-
 function setPosition(pos) {
-    if (posChanging) {
-        return;
-    }
-    var oldval = initgui;
-    if (pos > songlength) {
-        pos = songlength;
-        pausePosTimer();
+    synced = false;
+    progressTimer.set(pos);
+    if (!play) {
+        // Set slider position directly as timer callback is not currently
+        // running.
+        $("#trackslider").val(pos).slider('refresh');
```

Comment:
I think this relates to the comments that I made on the fact that the timer has been patched.

The timer can't always be used to update the slider as it interferes with user events. 

But I could also just be being dense...